sigstore/_internal/oidc.py

```diff
@@ -0,0 +1,48 @@
+import jwt
+
+# From https://github.com/sigstore/fulcio/blob/b2186c01da1ddf807bde3ea8c450226d8e001d88/pkg/config/config.go#L182-L201  # noqa
+OIDC_ISSUERS = {
+    "https://accounts.google.com": {
+        "proof_claim": "email",
```

Comment:
Do we need to nest this dict in here? Can't it just map directly from the issuer to the proof?

Reply:
I was imagining we might need more per-issuer details eventually, but I agree that it's not necessary right now.

Reply:
Makes sense. I think we can cross that bridge when we come to it. If we need more issuer details, it might make more sense to have an `OIDCIssuer` abstract base class or something like that.